src/common/utils/gpo.h

```diff
@@ -116,6 +119,53 @@ namespace powertoys_gpo {
         }
     }
 
+    inline std::string getPolicyListValue(const std::wstring& registry_list_path, const std::wstring& registry_list_value_name)
+    {
+        // This function returns the value of an entry of an policy list. The user scope is only checked, if the list is not enabled for the machine to not mix the lists. If there is no value or no list is found in the registry we return an empty string.
+
+        HKEY key{};
+        DWORD reg_value_type = REG_SZ;
+        char* string_value{};
```

Comment:
```suggestion
        char string_value[100];
```

Try this

Reply:
not a nice solution, just pointing out where the problem is :)

Reply:
Bad. Then really need to revert it back to 1024. 🤔 Is there any better solution that is more flexible?

Reply:
https://github.com/microsoft/PowerToys/blob/main/src/modules/fancyzones/FancyZonesLib/VirtualDesktop.cpp#L141-L153

You can try something like this. First call to get buffer capacity needed and second to actually get the data

Reply:
Fixed now. I used a fixed length.